### PR TITLE
[ES]Translation of 02-2.md

### PR DESCRIPTION
**Pull request by andrescastro-itm**
Spanish translation of file 02-2.md for review

### CODE REVIEW

File: docs/es/week02/02-2.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
+---
+lang-ref: cap.02-2
+title: Calcular los gradientes para los módulos NN y los trucos prácticos para la propagación hacia atrás
+authors: Micaela Flores, Sheetal Laad, Brina Seidel, Aishwarya Rajan
+date: 3 February 2020
+---
```
Comment (Atcold) [inline docs/es/week02/02-2.md]: Missing
```
lang: es                      
translation-date:
translator:
```

**Comment by Atcold** [comment]
See #165.

**Comment by andrescastro-itm** [comment]
Thanks Alfredo, once I add the missing info, I should make a new pull request?

File: docs/es/week02/02-2.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+--- 
+
+<!--
+---
+lang-ref: ch.02-2
+title: Computing gradients for NN modules and Practical tricks for Back Propagation
+authors: Micaela Flores, Sheetal Laad, Brina Seidel, Aishwarya Rajan
+date: 3 February 2020
+--- 
+-->
```
Comment (Atcold) [inline docs/es/week02/02-2.md]:
```suggestion
```

**Pull request merged**